our_files/custom.js

```diff
@@ -1058,6 +1058,88 @@ var renameDueStatus = function() {
 };
 
 //Detail View Tasks -- ITEM_HOLD_LINK tasks
+/*
```

Comment:
Added comments here as a model: Purpose, URL to see in production, Python test that tests this functionality if it exists